packages/squiggle-lang/src/rescript/Reducer/Reducer_ErrorValue.res

```diff
@@ -62,3 +64,20 @@ let errorToString = err =>
   | RENeedToRun => "Need to run"
   | REOther(msg) => `Error: ${msg}`
   }
+
+let fromException = exn =>
+  switch exn {
+  | ErrorException(e) => e
+  | Js.Exn.Error(e) =>
+    switch Js.Exn.message(e) {
+    | Some(message) => REOther(message)
+    | None =>
+      switch Js.Exn.name(e) {
+      | Some(name) => REOther(name)
+      | None => REOther("Unknown error")
```

Comment:
Maybe we could flag these a bit more; like say, "Unknown error: Reducer_ErrorValue 1"

Reply:
Maybe a better message here is, "JS Error, with Error message and name not found"